---
title: 例外のメッセージ 2 を完了 |Microsoft ドキュメント
ms.custom: ''
ms.date: 06/08/2017
ms.prod: biztalk-server
ms.reviewer: ''
ms.suite: ''
ms.tgt_pltfrm: ''
ms.topic: article
helpviewer_keywords:
- exceptions, completing the flow
ms.assetid: 9cae28f7-6eec-42e1-91ae-ce4db582b90d
caps.latest.revision: 7
author: MandiOhlinger
ms.author: mandia
manager: anneta
ms.openlocfilehash: 6a10920f0b039d6c790e74de6c462a649d4ddf35
ms.sourcegitcommit: cb908c540d8f1a692d01dc8f313e16cb4b4e696d
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 09/20/2017
---
# <a name="completing-the-exception-message"></a>例外完了メッセージ
例外のフローを完了するには、作成、**送信**メッセージし、エラー メッセージ _ 3 を監視対象フォルダーにリダイレクトします。  
  
 ![](../core/media/jdeoneworld-02.gif "JdeOneWorld_02")  
  
## <a name="see-also"></a>参照  
 [例外のキャッチ ブロックを追加する方法](../core/how-to-add-a-catch-exception-block4.md)   
 [BizTalk Server 例外処理の使用](../core/using-biztalk-server-exception-handling1.md)